src/NServiceBus.AcceptanceTests/Routing/when_replying_to_a_message_sent_to_specific_instance.cs

```diff
@@ -0,0 +1,86 @@
+﻿namespace NServiceBus.AcceptanceTests.ScaleOut
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.Routing;
+    using NUnit.Framework;
+
+    [Explicit("We don't yet have a way to inspect on which receiver a message has been received.")]
```

Comment:
Receiver has a specific instance ID
Sender sends him a message using instance specific routing.
That message arrives in the instance-specific queue (Receiver-XYZ)
The receiver replies to the sender
The `reply-to` address should be set to `Receiver-XYZ` because this was the queue the message has been receiver from. Instead it is set to `Receiver` because this is the `LocalAddress` of this endpoint.